server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -395,6 +398,15 @@ private GitWithAuth openGit(File workDir) throws Exception {
         }
     }
 
+    private static String generateCommitDetail(String abbrId, RevCommit headCommit) {
+        final String newLine = System.lineSeparator();
+        return "The commit information of " + abbrId + ": " + newLine +
+               "Author: " + headCommit.getAuthorIdent().getName() + " <" +
+                       headCommit.getAuthorIdent().getEmailAddress() + "> " + newLine +
+               "Date: " + headCommit.getAuthorIdent().getWhen() + newLine +
```

Comment:
How about aligning?
```suggestion
               "Author: " + headCommit.getAuthorIdent().getName() + " <" +
                       headCommit.getAuthorIdent().getEmailAddress() + "> " + newLine +
               "Date:   " + headCommit.getAuthorIdent().getWhen() + newLine +
```

Reply:
Let's extract as `final PersonIdent authorIdent = headCommit.getAuthorIdent();` because `getAuthorIdent()` has some cost internally to parse.

Reply:
Nice find 👍